src/components/CardTalent/index.tsx

```diff
@@ -1,11 +1,30 @@
 import styled from "styled-components";
 import COLORS from "../../util/colors";
+import { Talent } from "../../types";
+import Icon from "../Icon";
+import { useMemo } from "react";
+import { rawDescriptionToReact } from "../../util/descriptionParser";
 
 interface Props {
-  title: string;
-  description: string;
+  talent: Talent;
 }
 
+const CardTalent = ({ talent }: Props) => {
+  const memoizedDescription = useMemo(() => rawDescriptionToReact(talent.rawDescription), [talent.rawDescription]);
+  return (
+    <TalentCarouselCard>
+      <h1 style={{ color: COLORS.foregroundPrimary }}>{talent.name}</h1>
+      <IconWrapper>
+        <Icon path={talent.iconLink} width={64} />
+      </IconWrapper>
+      <BasicInfoContainer>
+        <div>Level {talent.requiredLevel}</div>
+      </BasicInfoContainer>
+      <DescriptionContainer>{memoizedDescription}</DescriptionContainer>
+    </TalentCarouselCard>
```

Comment:
/nit: There's some box-shadow and border-radius on the PowerCarouselCard that would be good to bring across here (or ideally re-use)